GOVERNANCE.md

```diff
@@ -0,0 +1,49 @@
+# Project Governance (Very Minimal Governance model)
+
+This open source project is managed by a Steering Committee composed of the maintainers of this project. Maintainers are
+defined as individuals with full commit access to the project repositories.
+
+## Steering Committee
+
+The Steering Committee will be responsible for oversight of all technical, project, approval, and policy matters for the
+project. This notably includes brand and trademark management.
+
+The Steering Committee members are listed in the MAINTAINERS.md file in the repository. New maintainers (and accordingly,
+Steering Committee members) may be added or removed as outlined in [RESPONSIBILITIES.md](./RESPONSIBILITIES.md). The
+Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. If the Steering Committee
+Chair is removed from the Committee (or the Chair steps down from that role), it is the responsibility of the Steering
+Committee to appoint a new Chair.
+
+The Steering Committee may, at its discretion, add or remove members who are not maintainers.
+
+## Voting
+
```

Comment:
What are decisions that will be voted? Do we need to discuss everything publicly or can we discuss internally and publish only the final consensus?

Reply:
The committee only votes when it can't reach consensus.  So yes, the steering committee can discuss amongst themselves and only go to a public vote if they don't agree.  

As the project expands, the steering committee can also agree to set up sub-committees - for example a product management sub-committee to own the roadmap.